data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+        },
+        description: {
+            content: "They pressure governments, investors and insurers to stop supporting extraction and burning of coal in Europe. They track all planned and running [coal plants in the EU](https://beyond-coal.eu/data/) with data on health impact and emissions"
+            +" Europe Beyond Coal is supported by HEAL, WWF, Foundation “Development YES – Open-Pit Mines NO”, Greenpeace, CEE Bankwatch, ClientEarth and CAN Europe",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Bankwatch",
+            link: "https://bankwatch.org/",
+            keywords: ["finance", "divestment", "watchdog", "ngo"],
+            label: "bank-watch",
+        },
+        description: {
+            content: "They investigate the impacts of public finance on communities and help protect their rights and livelihoods."
+            + " Bankwatch monitors the European Bank for Reconstruction and Development, the European Investment Bank, the Asian Infrastructure Investment Bank and the EU in creating future budgets."
+            + " They meet their representatives to make local voices heard to fight for more sustainable and fair investments."
+            + " In addition Bankwatch works on the People's Budget, a proposed budget for the EU that (among others) aims at ending funding for fossil fuels.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Development YES - Open Pit Mines NO",
+            link: "https://rozwojtak-odkrywkinie.pl",
+            keywords: ["coal", "grassroots", "ngo"],
+            label: "development-yes-open-pit-mines-no",
+        },
+        description: {
+            content: "This movement aims to prevent plans to build new lignite open-pit mines and transform the Polish economy to higher efficiency and renewables."
+            + " From their website its unclear if it's still active and whether the initiative has their own actions or is rather a network of other initiatives.",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "REDD.plus",
+            link: "https://www.redd.plus/",
+            keywords: ["offsetting", "reforestation", "deforestation", "private-sector", "public-sector"],
+            label: "redd-dot-plus",
+        },
+        description: {
+            content: "REDD stands for Reducing emissions from deforestation and forest degradation and is part of the UN Framework Convention on Climate Change. It's a private-public initiative, seemingly setting up a registry to allow buying credits that support protection of forest areas."
+            + " The signup link doesn't work and there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
```

Comment:
Either delet REDD or we just agree that this topics like monitoring and 'green capitalism' are highly under critism so we do not show the critics on wikipedia. Other NGOs also have been critized a lot in the past and we did not mentioned it.

Reply:
I think it's okay if we take a position and also selectively share criticism. Sharing no criticism and sharing all criticism would be false balance.
We can discuss deplattforming initiatives we don't like or leaving out specific classes of initiatives.

I'd rather keep initiatives on and comment on them.

In this case it's highly relevant as it has been suggested by the [Giving Pledge](https://www.idinsight.org/givinggreen) so we can't hide it without explanation.

Reply:
ok let's leave everything like it is. Rethought it and liked it :)